app/_data/docs_nav_konnect.yml

```diff
@@ -357,6 +357,13 @@
           absolute_url: true
           generate: false
           target_blank: true
+    - text: Konnect Search API (Beta)
```

Comment:
@cloudjumpercat we're aiming to alphabetize the APIs in the nav, so this should move up please and thanks.